source/scale/scaling-for-enterprise.rst

```diff
@@ -6,85 +6,40 @@ Scaling for Enterprise
+Tests were defined by configuration of the actions executed by each simulated user (and the frequency of these actions). The coordinator metrics define a health system under load. Full testing methodology, configuration, and setup is available. Sign up at `Mattermost Community <https://community.mattermost.com/>`__ and join the `Developers: Performance channel <https://community.mattermost.com/core/channels/developers-performance>`__.
 
-To scale to 100,000 users and above, we recommended using `the Mattermost open source load testing framework <https://github.com/mattermost/mattermost-load-test-ng>`__ to simulate usage of your system at full scale.
+Mattermost load testing tools
+-----------------------------
 
-.. csv-table::
-    :header: "Feature", "Self-Hosted", "Amazon AWS", "Azure"
+Mattermost provides a set of tools written in Go to help profiling Mattermost under heavy load, simulating real-world usage of a server installation at scale. The `Mattermost Load Test Tools <https://github.com/mattermost/mattermost-load-test-ng>`__ estimates the maximum number of concurrently active users the target system supports, and enables you to control the load to generate.
 
-    "Multi-region/data center support", "Yes", "Regions: 16", "Regions: 54"
-    "Auto scaling for Mattermost nodes", "Yes - using a solution like Kubernetes", "`AWS Auto Scaling <https://aws.amazon.com/ec2/autoscaling/>`__", "`Azure Autoscale <https://azure.microsoft.com/en-us/features/autoscale/>`__"
-    "Geographic based traffic routing", "Yes", "`Route 53 <https://aws.amazon.com/route53/>`__", "`Azure DNS <https://azure.microsoft.com/en-us/services/dns/>`__"
-    "Load balancing", "Yes", "`Elastic Load Balancer <https://aws.amazon.com/elasticloadbalancing/>`__", "`Azure Load Balancer <https://azure.microsoft.com/en-us/services/load-balancer/>`__"
-    "Multi-region, HA storage", "Yes", "`S3 <https://aws.amazon.com/s3/>`__", "?"
-    "Multi-region, HA MySQL", "Yes - using a solution like Galera", "`Aurora <https://aws.amazon.com/rds/aurora/>`__ / `RDS for MySQL <https://aws.amazon.com/rds/mysql/>`__", "`Azure MySQL <https://azure.microsoft.com/en-us/services/mysql/>`__"
-    "Multi-region, HA PostgreSQL", "Yes", "`Aurora <https://aws.amazon.com/rds/aurora/>`__ / `RDS for PostgreSQL <https://aws.amazon.com/rds/postgresql/>`__", "`Azure PostgreSQL <https://azure.microsoft.com/en-us/services/postgresql/>`__"
-    "Multi-region, HA elasticsearch", "Yes", "`Amazon Elasticsearch Service <https://aws.amazon.com/opensearch-service/>`__", "No"
+Visit the `Mattermost Load Test Tools <https://github.com/mattermost/mattermost-load-test-ng/tree/master/docs>`__ documentation on GitHub for details on getting started with the tools, and visit `GDoc <https://godoc.org/github.com/mattermost/mattermost-load-test-ng>`__ for code-specific documentation.
```

Comment:
`GDoc` reads like Google Docs to me. I also usually link to https://pkg.go.dev/github.com/mattermost/mattermost-load-test-ng directly, not godoc.